src/SourceBuild/tarball/content/tools-local/init-build.proj

```diff
@@ -96,6 +108,17 @@
                       Lines="$(PackageVersionsPropsContents)"
                       Overwrite="true" />
 
+    <!--
```

Comment:
This logic was moved to the end of the target to keep the previous source-built logic grouped.